src/components/ActionsMenu/Item/ActionsMenuItem.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import { TPAComponentsConsumer } from '../../TPAComponentsConfig';
+import styles from './ActionsMenuItem.st.css';
+import { Text } from '../../Text';
+
+export interface ActionsMenuItemProps {
+  prefixIcon?: React.ReactNode;
+  /** the item's content */
+  content: string;
+  /** the subtitle of the item */
+  subtitle?: string;
+  disabled?: boolean;
+  onClick(): void;
+}
+
+/** ActionsMenu */
+export class ActionsMenuItem extends React.Component<ActionsMenuItemProps> {
+  static displayName = 'ActionsMenu.Item';
+
+  render() {
+    const {
+      content,
+      prefixIcon,
+      subtitle,
+      disabled,
+      onClick,
+      ...rest
+    } = this.props;
+
+    return (
+      <TPAComponentsConsumer>
+        {({ mobile }) => (
+          <div
```

Comment:
Please refer to http://w3c.github.io/aria-practices/#menu for accessibility 
I think you shouldn't use `aria-pressed` as it is used for toggle buttons mostly, you should use `role="menu"`, set `aria-disabled` for disabled items, use a `ul` tag and more

Reply:
Add mentioned comments addressed, but I'm not sure what's behind "more". 
Please ping me if something else need to be changed